front/src/UI/organisms/AlarmChatMessagesBox/index.tsx

```diff
@@ -0,0 +1,41 @@
+import { useQuery } from '@apollo/client';
+import React, { useEffect } from 'react';
+import { AlarmChatMessage } from '../../molecules';
+import { GET_CURRENT_USERID } from './AlarmChatMessagesBoxQueries';
+
+export const AlarmChatMessagesBox = ({ ...props }) => {
+  const { subscribeToNewMessage, chatLog, chatIndex, chatUUID } = props;
+  useEffect(() => {
+    const unsubscribe = subscribeToNewMessage({
+      document: chatUUID,
+    });
+    return () => {
+      unsubscribe();
+    };
+  });
+  // 로그인 정보 가져오기
+  const { data, loading, error } = useQuery(GET_CURRENT_USERID);
+  if (loading) {
+    return <>Loading...</>;
+  }
+  if (error) {
+    console.log(error);
+    return <>ERROR</>;
+  }
+  return (
+    <>
+      {chatLog.map(({ index, type, userID, message, createdAt }) => {
+        type = type === 'notification' ? type : userID === data.me.userID ? 'ownerMessage' : 'message';
```

Comment:
```suggestion
       const type = type === 'notification' ? type : userID === data.me.userID ? 'ownerMessage' : 'message';
```
뭔가 제가 잘못 설정한지모르겠습니다. const가 없는데도 lint err가 나지 않은건가요? 음.. 별도로 type은 따로 다른 이름으로 선언하는게 좋아보입니다.